views/components/Sidebar/SidebarLogo.css

```diff
@@ -0,0 +1,12 @@
+.logo_container {
+  height: 98px;
+  width: 240px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 32px 16px 32px 16px;
+}
+.logo_image {
+  width: 197px;
+  height: 45px;
```

Comment:
do we need this? I think we can do something like `max-width: 100%` and `min-width: 100%` and let the image size itself.

Reply:
[465c369](https://github.com/AMP-SCZ/dpdash/pull/458/commits/465c36952e0f8176957c65d89dfea1c6b86ca296)

Reply:
this hasn't been resolved, do we need this height?